src/python/pants/init/engine_initializer.py

```diff
@@ -27,7 +27,7 @@
 from pants.engine.rules import SingletonRule
 from pants.engine.scheduler import Scheduler
 from pants.option.global_options import GlobMatchErrorBehavior
-from pants.scm.change_calculator import EngineChangeCalculator
+from pants.option.options_bootstrapper import OptionsBootstrapper
```

Comment:
This import looks unused. I believe that if `isort` weren't failing, this would show up as the next lint failure.

Reply:
It looks like it's being used in `L123` though

Reply:
Unless a symbol is literally referenced (spelled out) it doesn't need to be imported. But the CI can confirm.